js/core/clusters/clusters.js

```diff
@@ -53,6 +53,11 @@ define([
   return function(config) {
     var lock = false;
 
+    // getter for config.cluster
+    window.cluster = function() {
```

Comment:
This is better than before but I still consider co-existence of window.cluster() and config.cluster as a technical debt. It must be fixed for the long term.

If we **need** to have window.cluster[()], then config.cluster must be removed.